back/engines/free/email_campaigns/app/models/concerns/email_campaigns/schedulable.rb

```diff
@@ -37,5 +37,17 @@ def force_schedule_start_in_config_timezone
       ics.start_time = ics.start_time.in_time_zone(AppConfiguration.instance.settings('core', 'timezone'))
       self.ic_schedule = ics
     end
+
+    def schedule_multiloc
+      # We currently only support weekly schedules here.
+      weekly_day_key = "email_campaigns.schedules.weekly.#{schedule['rrules'][0]['validations']['day'][0]}"
+      hour_of_day = schedule['rrules'][0]['validations']['hour_of_day'][0]
+
+      AppConfiguration.instance.settings('core', 'locales').each_with_object({}) do |locale, result|
+        I18n.with_locale(locale) do
+          result[locale] = I18n.t(weekly_day_key, atHour: "@ #{hour_of_day}:00")
+        end
+      end
+    end
```

Comment:
I feel it makes more seems to have this method in `Schedulable`, but it's easy to move into the serializer if this is preferred.